src/utils/validation.ts

```diff
@@ -130,6 +111,28 @@ export const validateCreatePayment = (requestArgs: PaymentRequest) => {
     return false
   }
 
+  if (requestArgs._sourceType === 'individual') {
```

Comment:
Will add tests for `'company'` as soon as we've nailed down the desired behavior